### PR TITLE
Add community button to frontend menu

### PR DESCRIPTION
**Pull request by gimoAI**
This PR adds a community button as discussed in https://github.com/asreview/asreview/issues/1209

community_button

The chosen icon is unavailable in the 5.0.1 version of @miu/icons-material package, hence the update to the newest version.

### CODE REVIEW

File: asreview/webapp/src/Components/DrawerItemContainer.js [hunk 1]
```diff
@@ -19,12 +19,12 @@ import {
 } from "@mui/material";
 
 import { styled } from "@mui/material/styles";
-import { Help, Payment, Settings } from "@mui/icons-material";
+import { Help, Payment, Settings, Diversity3 } from "@mui/icons-material";
```
Comment (J535D165) [inline asreview/webapp/src/Components/DrawerItemContainer.js:22]: Can you order the imports alphabetically?

File: asreview/webapp/src/Components/DrawerItemContainer.js [hunk 2]
```diff
@@ -19,12 +19,12 @@ import {
...
 
 import { DrawerItem, ElasGame } from "../Components";
 
 import { ProjectAPI } from "../api/index.js";
-import { donateURL, projectModes, projectStatuses } from "../globals.js";
+import { donateURL, communityURL, projectModes, projectStatuses } from "../globals.js";
```
Comment (J535D165) [inline asreview/webapp/src/Components/DrawerItemContainer.js:27]: Same

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks. And welcome to the project!

**Review by terrymyc — COMMENTED** [review]
Thanks! Can you update the corresponding screenshots in the documentation? Please follow the development [guideline](https://github.com/asreview/asreview/blob/master/DEVELOPMENT.md#documentation).

**Comment by J535D165** [comment]
Very nice @gimoAI !

**Pull request merged**